### ISSUE TITLE
Request: Port for Fabric

### ISSUE THREAD

**Issue by vico93** [Lemonszz/Totem-Expansion#23]
First of all, thanks a lot for this mod!

Currently i switched modloaders, from Forge to Fabric for 1.16/1.17+.

Your mod is one of the few without a port or a equivalent mod for Fabric.

I requested for a port or an equivalent mod for Fabric (because, since this mod wasnt updated past 1.13, i thought it was inactive), but a member pointed out you are working with Fabric as well, so the member told me to try asking directly you about a Fabric port.

PS.: Just want to point there is equivalents for Baubles for Fabric 1.16+: [Curios API Fabric Edition](https://www.curseforge.com/minecraft/mc-mods/curios-fabric) and [Trinkets](https://www.curseforge.com/minecraft/mc-mods/trinkets-fabric).

Sorry if i'm bothering you, and thanks again for your mod!

**Comment by florensie** [comment]
FYI: Curios is going away with 1.17 on Fabric. The devs from both libraries are working together to rewrite Trinkets for 1.17. Both libraries have their strengths right now (I've used both) but it doesn't make much sense to have two.

**Comment by Lemonszz** [comment]
Yeah it's totally on the table for the future. If I was porting a mod I've made in the past forward, it would likely be this one.

Can't give any time estimates or anything though.

**Comment by vico93** [comment]
Hello, any news about this?

For now i think [Trinkets](https://modrinth.com/mod/trinkets) would be a better API if a port to Fabric/Quilt is desirable.

Of course, if @Lemonszz can't port it anymore, can other modder do it? I can ask on Fabric's Discord requests channel if this is the case.

**Comment by Lemonszz** [comment]
I'm not likely to touch this mod again.

The license is MIT so anyone is welcome to take anything from this mod or port it to whatever.
